### ISSUE TITLE
bug error

### ISSUE THREAD

**Issue by luokeli59** [dbgee/navicat-crack#1]
#  在执行此命令时报错：
./navicat-patcher ~/navicat16-premium-en-patched
#  报错信息如下：

***************************************************
*       navicat-patcher by @DoubleLabyrinth       *
*               version: 16.0.7.0-3               *
***************************************************

[+] Try to open libcc.dll ... OK!

[-] patch_solution_since<16, 0, 7, 0>: Vftable for CSRegistrationInfoFetcher_LINUX is not found.
[-] patch_solution_since<16, 0, 7, 0>: This patch solution will be suppressed.
[-] .\navicat-patcher\elf64_interpreter.cpp:570 ->
    File offset(0xffff80f3d35cd008) doesn't have corresponding virtual address.

# 不知道什么原因导致的？

**Comment by luokeli59** [comment]
# os version
cat /etc/os-release 
PRETTY_NAME="Deepin 20.9"
NAME="Deepin"
VERSION_ID="20.9"
VERSION="20.9"
VERSION_CODENAME="apricot"
ID=Deepin
HOME_URL="https://www.deepin.org/"
BUG_REPORT_URL="https://bbs.deepin.org/"


# kernel version
uname -r
5.15.77-amd64-desktop

#  depend
sudo apt install cmake libfmt-dev libssl-dev rapidjson-dev

**Comment by bsdrazor** [comment]
HI! Is anyone still maintain this project? Patch/Keygen does'nt work on Linux (Ubuntu) navicat version 16.2.2
